### DISCUSSION TITLE
USB C port accessible for gcode files?

### DISCUSSION THREAD

**Discussion by jaymcd** [OpenNeptune3D/OpenNept4une#166] [General]
Can the USB C port be used for storage of gcode files?

**Comment by halfmanbear** [comment]
Unfortunately not this is a serial terminal port only

**Comment by jaymcd** [comment]
Thanks for the super quick response! What about the USB-A port. If I add a small hub I should be able to use both wireless adapater and a USB drive ... ?

**Reply by halfmanbear** [reply]
Yes you can use a hub although make sure its supported by Linux without drivers, also it's a usb 3.0 port so best getting a hub that supports it too

**Reply by jaymcd** [reply]
Thanks. i have my usb hub with wifi adapter and a usb drive. They are recognized and I managed to map the usb drive with symbolic link to the gcodes area but when I try to use a file from this area the print starts, heats up and they stays in limbo and doesn't complete....

**Comment by halfmanbear** [comment]
Did you set that symlink up via the option in `opennept4une`  main script?

**Reply by jaymcd** [reply]
no. I think I have it resolved now. Turns out that my gcode/ profile may have been corrupted as I upgraded to Cura 5.7 from the beta...
